widgetssdk/src/main/java/com/glia/widgets/chat/adapter/holder/OperatorMessageViewHolder.java

```diff
@@ -80,12 +81,20 @@ private void addSingleChoiceCardView(
                 0
         );
         contentLayout.addView(singleChoiceCardView, params);
+
+        itemView.setContentDescription(item.content);
     }
 
     private void addMessageTextView(OperatorMessageItem item) {
         TextView contentView = getMessageContentView();
         contentView.setText(item.content);
         contentLayout.addView(contentView);
+
+        if (!TextUtils.isEmpty(item.operatorName)) {
+            itemView.setContentDescription(context.getString(R.string.glia_chat_operator_name_message_content_description, item.operatorName, item.content));
```

Comment:
please break the long lines to improve readability

Reply:
fixed